measure/src/measure.rs

```diff
@@ -84,6 +84,18 @@ impl Measure {
     }
 }
 
+#[macro_export]
+macro_rules! measure {
+    ($val:expr, $name:tt, $var1:ident, $var2:ident $(,)?) => {
+        let mut $var2 = Measure::start($name);
+        let $var1 = $val;
+        $var2.stop();
+    };
+    ($val:expr, $var1:ident, $var2:ident $(,)?) => {
+        measure!($val, "", $var1, $var2)
+    };
+}
```

Comment:
You just need to add a scope `{}` so that the `let` is allowed:

```suggestion
macro_rules! measure {
    ($val:expr, $name:tt $(,)?) => {{
        let mut measure = $crate::measure::Measure::start($name);
        let result = $val;
        measure.stop();
        (result, measure)
    }};
    ($val:expr) => {
        measure!($val, "")
    };
}
```

Notes:
1. I  removed the trailing comma `,` from the no-name version since it only has one parameter.
2. The call to `Measure::start()` needed to be hygenic. See https://gist.github.com/Kestrer/8c05ebd4e0e9347eb05f265dfb7252e1#declarative-macros-use-crate for more info.

Reply:
Ahh I see, looks good!

Reply:
Done!